pyccel/parser/syntactic.py

```diff
@@ -491,17 +492,14 @@ def _visit_UnaryOp(self, stmt):
             Func = PyccelMinus
 
         elif isinstance(stmt.op, ast.Invert):
-
-            errors.report(PYCCEL_RESTRICTION_UNARY_OPERATOR,
```

Comment:
You can remove this error message from the list of possible errors in `pyccel/errors/messages.py`

Reply:
Yes, this should be done within this PR.